chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/manager/ChangeRequestManagerImpl.java

```diff
@@ -121,26 +123,15 @@ public ChangeRequest createChangeRequest(final ChangeRequest cr)
     @PostAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).CHANGE_REQUEST, "
             + "T(gov.healthit.chpl.permissions.domains.ChangeRequestDomainPermissions).GET_BY_ID, returnObject)")
     public ChangeRequest getChangeRequest(final Long changeRequestId) throws EntityRetrievalException {
-        ChangeRequest cr = new ChangeRequest();
-        cr = changeRequestDAO.get(changeRequestId);
-        return populateChangeRequestData(cr);
+        return changeRequestDAO.get(changeRequestId);
     }
 
     @Override
     @Transactional(readOnly = true)
-    @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).CHANGE_REQUEST, "
-            + "T(gov.healthit.chpl.permissions.domains.ChangeRequestDomainPermissions).GET_ALL)")
+    @PostFilter("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).CHANGE_REQUEST, "
```

Comment:
Should we still have the PreAuthorize here as well so that users who would never be allowed to see change requests get an Access Denied error rather than an empty list (which is what I think the postfilter will do)?